templates/adspot/handlers/manage.ts

```
@@ -0,0 +1,87 @@
+}): Promise<BuildFrameData> {
+    const isOwner = body.fid.toString() === config.owner
+    const isWinner = body.fid.toString() === storage.currentAd.winner
+    const isBidder = storage.bids.some((bid) => bid.bidder === body.fid.toString())
+
+    if (isOwner) {
+        return handleOwnerActions(body, config, storage)
+    } else if (isWinner) {
+        return handleWinnerActions(body, storage)
+    } else if (isBidder) {
+        return handleBidderActions(body, config, storage, params)
+    }
```

Biome

[error] This else clause can be omitted because previous branches break early.

Unsafe fix: Omit the else clause.


(lint/style/noUselessElse)